src/posthog-core.js

```diff
@@ -1527,6 +1527,25 @@ PostHogLib.prototype.clear_opt_in_out_captureing = function (options) {
     this.clear_opt_in_out_capturing(options)
 }
 
+PostHogLib.prototype.sentry_integration = function (_posthog) {
+    this.setupOnce = function (addGlobalEventProcessor) {
```

Comment:
Is `this.setupOnce` intentional? what does it achieve?

Reply:
Yes, it's something that Sentry calls. I'll add a comment to clarify